src/components/InputRadio/InputRadio.tsx

```diff
@@ -107,7 +117,7 @@ const InputRadio = forwardRef(
     }, [isGroupStyle, addonProps]);
 
     return (
-      <div className="bsds-field">
+      <div className={`bsds-field ${className || ''}`}>
```

Comment:
i've been thinking of this as a way to adjust vertical spacing, so thought it would make sense to exclude this file since we can never have a single radio button and just apply spacing adjustments to the radio group component, but it may be worth asking catie if we want to give folks the ability to add extra classes on individual radio buttons.

if we want to keep this, we'll need to check out why it isn't showing as a control in SB.

Reply:
Catie said no, removing this 👍